rld/unit_tests/rld/TestXfxResolver.cpp

```diff
@@ -42,6 +42,7 @@ createFragmentWithReferenceTo(CompilationBuilder::Transaction &T,
   DataSection.data.emplace_back(uint8_t{0});
   DataSection.xfixups.emplace_back(Strings.add(T, Name), // name
                                    0,                    // relocation_type
+                                   false,                // is_weak
```

Comment:
I’m working on the rld side of this feature ATM. Can this modification be removed from this change?

Reply:
If I remove this change, the build server will fail the check-all since the  generates solution includes the rld project.

Reply:
Are you sure? The pstore code retains the original 4 argument external_fixup ctor specifically to avoid this problem.

Reply:
Sorry, forget that. I have removed the change.